internal/cmd/list-alerts.go

```diff
@@ -0,0 +1,262 @@
+// Copyright © 2020 The Homeport Team
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package cmd
+
+import (
+	"strings"
+	"time"
+
+	"github.com/PagerDuty/go-pagerduty"
+	"github.com/gonvenience/bunt"
+	"github.com/homeport/pd/internal/pd"
+	"github.com/spf13/cobra"
+)
+
+// currentShiftCmd represents the get command
+var listAlertsCmd = &cobra.Command{
+	Use:   "list-alerts",
+	Args:  cobra.ExactArgs(0),
+	Short: "Lists all alerts",
+	Long:  `Lists all alerts in a specified time period`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+
+		client, err := pd.CreatePagerDutyClient()
+		if err != nil {
+			return err
+		}
+
+		id, err := cmd.Flags().GetString("id")
+		if err != nil {
+			return err
+		}
```

Comment:
My bad, we never touched on this topic before. There are multiple ways to do flags in Go CLIs. Since we use `cobra`, I would ask you to migrate the flags to the `cobra` style: https://github.com/spf13/cobra#working-with-flags

We should do a pairing session on this.